httpx/met.go

```diff
@@ -0,0 +1,36 @@
+package httpx
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/bsm/rucksack/met"
+	"github.com/go-chi/chi/middleware"
+)
+
+var metMiddleware = middleware.RequestLogger(metFormat{})
+
+type metFormat struct{}
+
+func (metFormat) NewLogEntry(r *http.Request) middleware.LogEntry { return metEntry{Request: r} }
+
+type metEntry struct{ *http.Request }
+
+func (e metEntry) Write(status, _ int, elapsed time.Duration) {
+	metric := "http.request"
+	tags := append(make([]string, 0, 2),
+		"status:"+strconv.Itoa(status),
+	)
+	if status != 404 {
+		tags = append(tags, "path:"+e.URL.Path)
+	}
+
+	met.RatePerMin(metric, tags).Update(1)
+	met.Timer(metric+".time", tags).Update(elapsed)
```

Comment:
Really nitpicking, but we could have:

```
const reqMetric = "http.request"
const reqTimeMetric = reqMetric + ".time" // I think this should be valid in Go, or even just "http.request.time" here
```

Probs, would generate slightly less garbage :slightly_smiling_face: (unless Go optimises it properly, which I doubt)